pyteal/ast/subroutine.py

```diff
@@ -76,37 +78,176 @@ def __init__(
         self.declaration: Optional["SubroutineDeclaration"] = None
         self.__name = self.implementation.__name__ if nameStr is None else nameStr
 
+    @staticmethod
+    def check_if_ABI_class(arg_type: Type) -> bool:
```

Comment:
```suggestion
    def is_ABI_class(arg_type: Type) -> bool:
```
recommend renaming because `check_` sounds like some assertion is going to be made

Reply:
👍